internal/provider/resource_vpc_firewall_rules.go

```diff
@@ -527,12 +528,13 @@ func newVPCFirewallRulesModel(rules []oxide.VpcFirewallRule) ([]vpcFirewallRules
 
 	for _, rule := range rules {
 		m := vpcFirewallRulesResourceRuleModel{
-			Action:       types.StringValue(string(rule.Action)),
-			Description:  types.StringValue(rule.Description),
-			Direction:    types.StringValue(string(rule.Direction)),
-			ID:           types.StringValue(rule.Id),
-			Name:         types.StringValue(string(rule.Name)),
-			Priority:     types.Int64Value(int64(rule.Priority)),
+			Action:      types.StringValue(string(rule.Action)),
+			Description: types.StringValue(rule.Description),
+			Direction:   types.StringValue(string(rule.Direction)),
+			ID:          types.StringValue(rule.Id),
+			Name:        types.StringValue(string(rule.Name)),
+			// We can safely dereference rule.Priority as it's a required field
+			Priority:     types.Int64Value(int64(*rule.Priority)),
```

Comment:
I was thinking about using `types.XXPointerValue` but there's no such function for `int`, only `int32` and `int64` and we'd have to dereference our pointer anyway to cast.

Reply:
yeah :(